docs/develop/use-sso-to-get-office-signed-in-user-token.md

```diff
@@ -0,0 +1,346 @@
+
+# [Visual Studio 2019](#tab/vs2019)
+
+1. Open the **Home.js** file.
+1. Replace the entire contents of the file with the following code.
+
+   ```javascript
+   (function () {
+     "use strict";
+
+     // The initialize function must be run each time a new page is loaded.
+     Office.initialize = function (reason) {
+       $(document).ready(function () {
+         $("#getIDToken").click(getIDToken);
+       });
+     };
+
+     async function getIDToken() {
+       try {
+         let userTokenEncoded = await OfficeRuntime.auth.getAccessToken({
+           allowSignInPrompt: true,
+         });
+         let userToken = jwt_decode(userTokenEncoded);
+         document.getElementById("userInfo").innerHTML =
+           "name: " +
+           userToken.name +
+           "<br>email: " +
+           userToken.preferred_username +
+           "<br>id: " +
+           userToken.oid;
+         console.log(userToken);
+       } catch (error) {
+         document.getElementById("userInfo").innerHTML =
+           "An error occurred. <br>Name: " +
+           error.name +
+           "<br>Code: " +
+           error.code +
+           "<br>Message: " +
+           error.message;
+         console.log(error);
+       }
+     }
+   })();
+   ```
+
+1. Save the file.
+
+# [yo office](#tab/yooffice)
+
+1. Open the **src/taskpane/taskpane.js** file.
+1. Replace the entire contents of the file with the following code.
+
+   ```javascript
+   import jwt_decode from "jwt-decode";
+
+   Office.onReady((info) => {
+     if (info.host === Office.HostType.Excel) {
+       document.getElementById("getIDToken").onclick = getIDToken;
+     }
+   });
+
+   async function getIDToken() {
+     try {
+       let userTokenEncoded = await OfficeRuntime.auth.getAccessToken({
+         allowSignInPrompt: true,
+       });
+       let userToken = jwt_decode(userTokenEncoded);
+       document.getElementById("userInfo").innerHTML =
+         "name: " +
+         userToken.name +
+         "<br>email: " +
+         userToken.preferred_username +
+         "<br>id: " +
+         userToken.oid;
+       console.log(userToken);
+     } catch (error) {
+       document.getElementById("userInfo").innerHTML =
+         "An error occurred. <br>Name: " +
+         error.name +
+         "<br>Code: " +
+         error.code +
+         "<br>Message: " +
+         error.message;
+       console.log(error);
+     }
+   }
+   ```
+
+1. Save the file.
+
+---
+
+## Run the add-in
+
+# [Visual Studio 2019](#tab/vs2019)
+
+1. Choose **Debug > Start Debugging**, or press **F5**.
+
+# [yo office](#tab/yooffice)
+
+Run `npm start` from the command line.
+
+---
+
+1. When Excel starts, sign in to Office with the same tenant account you used to create the app registration.
+1. On the **Home** ribbon, choose **Show Taskpane** to open the add-in.
+1. In the add-in's task pane, choose **Get ID token**.
+
+The add-in will display the name, email, and ID of the account you signed in with.
```

Comment:
```suggestion
The add-in displays the name, email, and ID of the account you signed in with.
```
This result should be a paragraph under the last step so I added 4 spaces to indent. You can remove the blank line before it, for clarity.